api/src/pcapi/static/backofficev3/js/addons/pc-confirm-modal.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Store an input initial state and restore if modal gets dismissed.
+ *
+ * We have two behavior:
+ *
+ * 1. checkbox: will store the `checked` as initial value
+ * 1. other inputs: will store the `value` as initial value
+ *
+ * You must set `[data-confirm-modal]` attribute and set a not a generic css selector that target the `.modal`
```

Comment:
```suggestion
 and set a not a generic
```
Il y a un problème dans cette phrase ;)